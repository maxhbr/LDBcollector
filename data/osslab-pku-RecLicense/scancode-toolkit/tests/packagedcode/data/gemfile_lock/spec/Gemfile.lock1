  specs:
    user-lib (2.1.0)
