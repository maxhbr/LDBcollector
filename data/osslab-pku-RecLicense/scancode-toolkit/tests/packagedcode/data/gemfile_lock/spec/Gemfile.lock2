  specs:
    user-lib (2.1.0)
      multi_json
