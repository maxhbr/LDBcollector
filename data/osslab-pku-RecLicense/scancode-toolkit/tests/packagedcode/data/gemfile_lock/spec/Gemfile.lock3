  specs:
    activesupport (3.2.17)
      i18n (~> 0.6, >= 0.6.4)
      multi_json (~> 1.0)
    addressable (2.3.6)
    allowy (0.4.0)
      activesupport (>= 3.2)
      i18n
    i18n (0.6.9)
    multi_json (1.9.2)
