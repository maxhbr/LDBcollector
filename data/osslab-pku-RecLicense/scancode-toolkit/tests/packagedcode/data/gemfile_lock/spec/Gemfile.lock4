  specs:
    activesupport (3.2.17)
      i18n (~> 0.6, >= 0.6.4)
      multi_json (~> 1.0)
    addressable (2.3.6)
    allowy (0.4.0)
      activesupport (>= 3.2)
      i18n
    clockwork (0.4.1)
      fluent-logger
    delayed_job (4.0.0)
      activesupport (>= 3.0, < 4.1)
    fluent-logger (0.4.7)
      msgpack (>= 0.4.4, < 0.6.0, != 0.5.3, != 0.5.2, != 0.5.1, != 0.5.0)
      yajl-ruby (~> 1.0)
    i18n (0.6.9)
      clockwork
    msgpack (0.5.8)
    multi_json (1.9.2)
    yajl-ruby (1.2.0)
