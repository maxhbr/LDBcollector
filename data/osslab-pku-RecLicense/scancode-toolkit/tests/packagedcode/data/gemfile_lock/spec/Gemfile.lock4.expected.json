[
  {
    "name": "activesupport",
    "version": "3.2.17",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      ">= 3.2",
      ">= 3.0",
      "< 4.1"
    ],
    "dependencies": {
      "activesupport@3.2.17": {
        "i18n@0.6.9": {
          "clockwork@0.4.1": {
            "fluent-logger@0.4.7": {
              "msgpack@0.5.8": {},
              "yajl-ruby@1.2.0": {}
            }
          }
        },
        "multi_json@1.9.2": {}
      }
    }
  },
  {
    "name": "i18n",
    "version": "0.6.9",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      "~> 0.6",
      ">= 0.6.4"
    ],
    "dependencies": {
      "i18n@0.6.9": {
        "clockwork@0.4.1": {
          "fluent-logger@0.4.7": {
            "msgpack@0.5.8": {},
            "yajl-ruby@1.2.0": {}
          }
        }
      }
    }
  },
  {
    "name": "multi_json",
    "version": "1.9.2",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      "~> 1.0"
    ],
    "dependencies": {
      "multi_json@1.9.2": {}
    }
  },
  {
    "name": "addressable",
    "version": "2.3.6",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "addressable@2.3.6": {}
    }
  },
  {
    "name": "allowy",
    "version": "0.4.0",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "allowy@0.4.0": {
        "activesupport@3.2.17": {
          "i18n@0.6.9": {
            "clockwork@0.4.1": {
              "fluent-logger@0.4.7": {
                "msgpack@0.5.8": {},
                "yajl-ruby@1.2.0": {}
              }
            }
          },
          "multi_json@1.9.2": {}
        },
        "i18n@0.6.9": {
          "clockwork@0.4.1": {
            "fluent-logger@0.4.7": {
              "msgpack@0.5.8": {},
              "yajl-ruby@1.2.0": {}
            }
          }
        }
      }
    }
  },
  {
    "name": "clockwork",
    "version": "0.4.1",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": "GEM",
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "clockwork@0.4.1": {
        "fluent-logger@0.4.7": {
          "msgpack@0.5.8": {},
          "yajl-ruby@1.2.0": {}
        }
      }
    }
  },
  {
    "name": "fluent-logger",
    "version": "0.4.7",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "fluent-logger@0.4.7": {
        "msgpack@0.5.8": {},
        "yajl-ruby@1.2.0": {}
      }
    }
  },
  {
    "name": "delayed_job",
    "version": "4.0.0",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [],
    "dependencies": {
      "delayed_job@4.0.0": {
        "activesupport@3.2.17": {
          "i18n@0.6.9": {
            "clockwork@0.4.1": {
              "fluent-logger@0.4.7": {
                "msgpack@0.5.8": {},
                "yajl-ruby@1.2.0": {}
              }
            }
          },
          "multi_json@1.9.2": {}
        }
      }
    }
  },
  {
    "name": "msgpack",
    "version": "0.5.8",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      ">= 0.4.4",
      "< 0.6.0",
      "!= 0.5.3",
      "!= 0.5.2",
      "!= 0.5.1",
      "!= 0.5.0"
    ],
    "dependencies": {
      "msgpack@0.5.8": {}
    }
  },
  {
    "name": "yajl-ruby",
    "version": "1.2.0",
    "platform": "ruby",
    "pinned": false,
    "remote": null,
    "type": null,
    "path": null,
    "revision": null,
    "ref": null,
    "branch": null,
    "submodules": null,
    "tag": null,
    "requirements": [
      "~> 1.0"
    ],
    "dependencies": {
      "yajl-ruby@1.2.0": {}
    }
  }
]